__main__.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import sys
 
+from bugz import BugzillaClient
```

Comment:
`bugz` in the `BugzillaClient` is an OK function name or alias, however, seem comment below about `Bugz` as a classname....

Reply:
Replied below, Ideally it would be Bugzilla but it does not work as there is already a bugzilla.py after installing the requirements